libs/server/Storage/Session/MainStore/MainStoreOps.cs

```diff
@@ -695,68 +709,75 @@ private unsafe GarnetStatus RENAME(ArgSlice oldKeySlice, ArgSlice newKeySlice, S
                             {
```

Comment:
this "hasETag" must always be true now, after your change above to make this the "else" of "!hasETag"